src/wandbot/retriever/chroma.py

```
@@ -0,0 +1,404 @@
+        else:
+            logger.info(f"Successfully confirmed collection metadata: {retrieved_metadata}")
+
+    @weave.op
+    @vector_store_retry_decorator
+    def query(self, 
+              query_texts: Optional[List[str]] = None, 
+              query_embeddings: Optional[List[float]] = None,
+              n_results: int = 1, 
+              filter: Optional[Dict[str, Any]] = None, 
+              where_document: Optional[Dict[str, Any]] = None,
+              include: List[str] = ['documents', 'metadatas', 'distances']
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)